othello/moderator/utils.py

```diff
@@ -0,0 +1,98 @@
+import enum
+import importlib.machinery
+import operator
+from functools import partial, reduce, wraps
+
+from .constants import *
```

Comment:
Wildcard imports are evil.

Reply:
Is this really necessary, `constants.py` is literally just a bunch of constant values

Reply:
Then can't you just do `from . import constants` and refer to `constants.X`?